e2e/testvalues/values.go

```diff
@@ -64,8 +64,7 @@ var IcadNewGenesisCommandsFeatureReleases = semverutil.FeatureReleases{
 	MinorVersions: []string{
 		"v0.5",
 	},
-}
-
+	
```

Comment:
I think we're missing a bracket here. This could be the reason of the build failures in the e2e tests.

Reply:
Oh, it's my mistake.

Reply:
Can you make these test run again? @crodriguezvega

Reply:
Yea. now this err is what I want to say!

Reply:
`useNewGenesisCommand = false`